packages/client/src/encryption/BrowserPersistentStore.ts

```diff
@@ -15,22 +15,22 @@ export default class BrowserPersistentStore implements PersistentStore<string, s
         this.store = createStore(this.dbName, 'GroupKeys')
     }
 
-    async has(key: string) {
+    async has(key: string): Promise<boolean> {
         const val = await this.get(key)
         return val == null
     }
 
-    async get(key: string) {
+    async get(key: string): Promise<any> {
         return get(key, this.store)
     }
```

Comment:
Do we know if the type returned is always a string here?
```suggestion
    async get(key: string): Promise<string> {
        return get(key, this.store)
    }
```